lib/jiff-client.js

```diff
@@ -35,6 +35,40 @@
     return x > y ? x : y;
   }
 
+  /**
+   * Check that an integer is prime. Used to safely set the modulus Zp.
+   * @method is_prime
+   * @memberof
+   * @instance
```

Comment:
The JSDOC here is a bit messed up, @method and @instance are used when things are part of a class.

@memberof requires that you specify which *namespace* this is part of, so that the docs for this are displaying in that namespace's page when the HTML docs are generated.

For example, you can see the JIFF namespaces on the right side panel here: https://multiparty.org/jiff/docs/jsdoc/

If you want to test what your changes to the docs, run `npm run-script gen-docs` in the terminal and open `<jiff directory>/docs/jsdoc/index.html` in the browser.